static/src/javascripts/projects/common/modules/sudoku/main.js

```diff
@@ -0,0 +1,41 @@
+/* jshint newcap: false */
+define([
+    'common/utils/$',
+    'common/utils/_',
+    'react',
+    'common/modules/sudoku/flatMap',
+    'common/modules/sudoku/grid'
+], function (
+    $,
+    _,
+    React,
+    flatMap,
+    Grid
+) {
+    return function () {
+        $('.js-sudoku').each(function (element) {
+            if (element.hasAttribute('data-sudoku-data')) {
```

Comment:
probably best to convert `element` to a bonzo object, so you get x-browser attribute methods for free

or use `map` instead of `each` (passes in `element` as a bonzo object, I think...)